index.html

```diff
@@ -1167,15 +1167,22 @@ <h3>Malicious Issuers and Verifiers</h3>
 tracking or sharing the presentation of particular credentials.
       </p>
       <p>
-A malicious [=issuer=] might intentionally attack group privacy by creating a
-unique status list per credential issued in order to establish a one-to-one mapping to track
-when a [=verifier=] processes a specific credential. Similarly, they could establish
-another a one-to-one mapping by using a different cryptographic key for every credential
-issued that is tracked in a status list.
+A malicious [=verifier=] might intentionally attack group privacy by sharing
+information from presented credentials with a malicious [=issuer=]. This
+sort of collusion is difficult to detect as it is typically performed in a
+secure communication channel between the [=issuer=] and the [=verifier=].
       </p>
       <p>
-A malicious [=verifier=] might intentionally attack group privacy by sharing
-information from presented credentials with a malicious [=issuer=].
+A malicious [=issuer=] might intentionally attack group privacy by creating a
+unique status list per credential issued in order to establish a one-to-one
+mapping to track when a [=verifier=] processes a specific credential. Similarly,
+they could establish another a one-to-one mapping by using a different
+cryptographic key for every credential issued that is tracked in a status list.
```

Comment:
```suggestion
cryptographic key for every issued credential that is tracked in a status list.
```

Reply:
If I understand what is meant correctly...
```suggestion
cryptographic key for each credential issued that is tracked by a given status list.
```